ClassTypes/PowerShell/EthicalHacking/Create-EthicalHackingLabplanAndLab.ps1

```diff
@@ -44,20 +57,23 @@ trap {
 #
 
 # Download AzLab module file, import, and then delete the file
+if (-not (Get-Module -ListAvailable -Name 'Az')) {
+    Import-Module Az -Force 
+}
 
-Import-Module Az.LabServices -Force
+$ClassName = "EthicalHacking"
 
 
 # Configure parameter names
-$rgName     = "$($ClassName)RG_" + (Get-Random)
-$labPlanName     = "$($ClassName)Acct_" + (Get-Random)
+$rgName     = "rg-$ClassName-$(Get-Random)"
+$labPlanName     = "lp-$ClassName$(Get-Random)"
 $labName    =  "$($ClassName)Lab"
 
 # Create resource group
 Write-Host "Creating resource group $rgName"
 $rg = New-AzResourceGroup -Name $rgName -Location $Location
     
-# Create Lab Account
+# Create Lab Plan
 Write-Host "Creating lab plan $labPlanName"
 $labPlan  = New-AzLabServicesLabPlan -ResourceGroupName $rgName -Name $labPlanName -Location $Location -AllowedRegion @($Location)
```

Comment:
It's important that we use 'ToLower()' for the RG name, there's a bug in the Labs portal on displaying labs and this is the workaround until the bug is fixed...